### DISCUSSION TITLE
匿名留言功能關閉說明

### DISCUSSION THREAD

**Discussion by jwlin** [jwlin/dgpa-job-site#1] [Announcements]
大家好！當初網站提供匿名留言，本意是讓使用者可以沒有壓力的發言，提供對於職缺有用的資訊。但目前留言區被程式化訊息洗板嚴重，已經失去此功能的意義，所以暫時關閉。這部份是我的疏忽，當初設計時應該要提供進階的過濾機制，例如 CAPTCHA 或整合第三方服務如 Disqus 等。之後若有空，會把留言功能完善後再開放。有任何意見想法也請在下方留言

**Comment by nedfrankie** [comment]
建議考慮實名制，每個帳號對機關評價有投十票評價權力，並希望每個帳號自制是曾待過機關、才給予評價。期待各位先進可以給後學一些小小的提醒，避免一些不思改進的機關，繼續禍害年輕公務員。

**Reply by DanielTsai117** [reply]
同上，這個人云亦云的時代，全匿名的方式在社群化跟謾罵攻擊的均衡早被破壞了～
建議實名制為宜

**Comment by jflin688** [comment]
I have some different opinions.
First, the Dgpajobs website is just like a news website, everyone who browse the delivering news should distinguish it is real or fake.
Second, I think no one will comment any job opportunity, if he or she should be do at the risk of being identified.  I still think to comment anonymously can gather more information about some specific job vacancy.  To limit any comment is not consistent with freedom of speech. 

Maybe, for some specific job vacancy, it should be controlled  to avoid  the same or similar comments.  It is marvelous and glorious, if the Dgpajobs website have the function of auto-deleting same or similar comments for  each job vacancy.

All above suggest to you, Dr. Lin.  Thank you providing us (civil servant in Taiwan) a good website to gather some information for some  job vacancy for a long time.

Please think about a civil servant, working in the branch agency of Ministry of Labor Affairs, commit suicide some days ago.  We need these comments,
